### PR TITLE
Tokenizer fix decode

### PR DESCRIPTION
**Pull request by ilya-lavrenov**

### CODE REVIEW

File: modules/custom_operations/user_ie_extensions/CMakeLists.txt [hunk 1]
```diff
@@ -2,6 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
```
Comment (likholat) [inline modules/custom_operations/user_ie_extensions/CMakeLists.txt:5]: Needs to update the instructions for separate module build: https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/custom_operations#build-custom-openvino-operation-extension-library

File: .ci/azure/linux.yml [hunk 2]
```diff
@@ -154,30 +154,37 @@ jobs:
...
-  - script: |
-      set -e
-      export PATH=$(WORK_DIR)/gradle-$(GRADLE_VER)/bin:${PATH}
-      . $(SETUPVARS) gradle clean build --info
-      for d in CPU HETERO:CPU; do
-        gradle test -Prun_tests -DMODELS_PATH=$(MODELS_PATH) -Ddevice=$d --info;
-      done
-    workingDirectory: $(REPO_DIR)/modules/java_api
-    displayName: 'Java tests'
+#  - script: |
```
[the remark this answers is not in the document]
Comment (likholat) [reply .ci/azure/linux.yml:157]: Oh, I see, fixed the test here: https://github.com/openvinotoolkit/openvino_contrib/pull/768

File: .ci/azure/windows.yml [hunk 3]
```diff
@@ -123,32 +122,51 @@ jobs:
         -DENABLE_INTEL_GPU=OFF ^
         -DENABLE_INTEL_GNA=OFF ^
         -DENABLE_CPPLINT=OFF ^
+        -DENABLE_SAMPLES=OFF ^
         -DENABLE_OV_ONNX_FRONTEND=ON ^
-        -DOPENVINO_EXTRA_MODULES=$(REPO_DIR)/modules ^
+        -DBUILD_FAST_TOKENIZERS=ON ^
```
Comment (ilya-lavrenov) [inline .ci/azure/windows.yml:127]:
```suggestion
```

**Review by mryzhov — APPROVED** [review]

**Pull request merged**